.gitignore

```diff
@@ -26,3 +26,15 @@ settings.json
 .pytest_cache/
 .cache/
 .venv/
+
+# Symlink from 'src/applications/bmqbrkr/run'
+src/applications/bmqbrkr/etc/etc
+
+# 'sim_cpp11_features.pl' backups
+src/groups/mwc/mwcex/mwcex_bindutil.h.bak
```

Comment:
wouldn't it be simpler to remove those backup files in that same script that calls sim_cpp11_features.pl ?

Reply:
We don't have access to the script that launches the `sim_cpp11_features.pl`, since it's done within `bde-tools`. I updated `.gitignore` with wildcard backup pattern which covers all the changes, and this is the simplest way.